plugins/services/src/js/pages/task-details/TaskDetail.js

```diff
@@ -29,7 +30,9 @@ const HIDE_BREADCRUMBS = [
   '/jobs/:id/tasks/:taskID/details',
   '/network/virtual-networks/:overlayName/tasks/:taskID/details',
   '/nodes/:nodeID/tasks/:taskID/details',
-  '/services/overview/:id/tasks/:taskID/details'
+  '/services/overview/:id/tasks/:taskID/details',
+  '/services/overview/:id/tasks/:taskID/logs',
```

Comment:
I think it should include optional params, like you specified further down
`logs(/:fileName)`